src/main/java/codeu/controller/FileUploadServlet.java

```diff
@@ -55,14 +66,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
       String uri = request.getRequestURI();
       String mediaOwner = null;
+      Media picture = null;
       if(uri.equals("/profile/FileUploadServlet")) {
         mediaOwner = (String)request.getSession().getAttribute("user");
+        picture = mediaStore.getProfilePicture(mediaOwner);
       }
       else if(uri.equals("/profileView/FileUploadServlet")) {
       	mediaOwner = (String)request.getSession().getAttribute("username");
+      	picture = mediaStore.getProfilePicture(mediaOwner);
+      }
+      else if(uri.equals("/chat/FileUploadServlet")) {
```

Comment:
Indentation is all over the places / not great, which hurts readability
Did you try https://google.github.io/styleguide/javaguide.html,?